api/controllers/index.js

```diff
@@ -69,3 +69,16 @@ exports.postComment = (req, res, next) => {
       next(err);
     });
 };
+exports.patchReviewVote = (req, res, next) => {
```

Comment:
I'd just say here to be careful with the naming convention - in theory this is an endpoint to patchReview and you could one day expand that out to be able to patch more than just the votes